core/src/test/java/org/elasticsearch/index/translog/TranslogTests.java

```diff
@@ -136,7 +137,7 @@ public void tearDown() throws Exception {
     }
 
     private Translog create(Path path) throws IOException {
-        return new Translog(getTranslogConfig(path), null);
+        return new Translog(getTranslogConfig(path), null, () -> SequenceNumbersService.UNASSIGNED_SEQ_NO);
```

Comment:
do we have a test that checks that the global checkpoint given by this supplier is properly persisted?